### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ombralunare**
What I did so far ...

### CODE REVIEW

**Review by CharlSteynberg — CHANGES_REQUESTED** [review]

File: index.html
Comment (ombralunare) [reply index.html:7]: done

File: script.js
Comment (ombralunare) [reply script.js:126]: please let me know how this is looking so far...
(haven't tested, waiting for ur approval/Review)

I haven't done the cards yet. Not too sure how to go about it. (is it similar to the code for "cardName"??) 
I haven't done any html yet - must i make the alarm button (loudButn) a 'submit' attr??

**Review by CharlSteynberg — COMMENTED** [review]
have you tested this?

**Comment by ombralunare** [comment]
I've tested it and it doesn't work. It doesn't store the created card or provide a new one... 

Will get on it

**Pull request merged**
